### PR TITLE
[New] add Ukrainian

### PR DESCRIPTION
**Pull request by olegfedak**
I work with Decap CMS, which runs with node-polyglot. I've noticed that Ukrainian pluralization is incorrect. So, I really need this update! -_-

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]
This will need tests.

**Comment by olegfedak** [comment]
@ljharb what sort of tests bisides `pretest` an `test` should I run?

**Comment by ljharb** [comment]
I think you misunderstood; your PR will need to add tests for Ukrainian.

File: index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ var russianPluralGroups = function (n) {
   if (lastTwo !== 11 && end === 1) {
     return 0;
   }
-  if (2 <= end && end <= 4 && !(lastTwo >= 12 && lastTwo <= 14)) {
+  if (end >= 2 && end <= 4 && !(lastTwo >= 12 && lastTwo <= 14)) {
```
Comment (ljharb) [inline index.js:43]: please revert these autofixes; they’re this way on purpose

**Review by ljharb — APPROVED** [review]
thanks!

**Pull request merged**